_posts/en/newsletters/2023-08-23-newsletter.md

```diff
@@ -174,7 +174,9 @@ Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
 Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
 [Bitcoin Inquisition][bitcoin inquisition repo].*
 
-- [Bitcoin Core #27460][] rpc: Add importmempool RPC FIXME:Murchandamus
+- [Bitcoin Core #27460][] adds a new `importmempool` RPC. The RPC will
+  load a `mempool.dat` file and attempt to add the loaded transactions
+  to its mempool.
```

Comment:
I went back and forth with myself, whether it is useful to mention the options to the broader audience, but I didn’t have a good idea how to make it relevant. My closest attempt was to say:

“The RPC has options to import some transaction metadata from the imported file that may be useful to mining pool operators or other power users.”

But I was only half convinced. Any thoughts?

Reply:
fwiw I don't really see those options being useful to someone who isn't a power user + trying to replicate something very specific on a test node

Reply:
If y'all are unconvinced, I'm unconvinced.  Dropping it from this week's newsletter.  If anyone has second thoughts, we can always cover it next week.

Reply:
Ah just to clarify, I didn't mean that the summary should be dropped, just that a description of the options didn't need to be added. I think the RPC in general is worth covering!